src/condor_tests/startd_xfer_bytes.run

```diff
@@ -0,0 +1,137 @@
+my $bb = "IIIIIIIIIIIIIII JJJJJJJJJJJJJJJ KKKKKKKKKKKKKKK LLLLLLLLLLLLLLL MMMMMMMMMMMMMMM NNNNNNNNNNNNNNN OOOOOOOOOOOOOOO PPPPPPPPPPPPPPP";
+for (my $i = 0; $i < (5*1024); $i++ ) { print DF $aa . "\t" . $bb . "\n"; }
+close(DF);
+
+my $in_ad  = '[TransferProtocol="BIB"; TransferSuccess=true; TransferTotalBytes=10000000;];';
+my $out_ad = '[TransferProtocol="BOB"; TransferSuccess=true; TransferTotalBytes=122200; TransferFileName="dummy"; TransferUrl="';
+   $out_ad .= "BOB://bar";
+   $out_ad .= '";]';
+
+# create transfer plugins
+#
+TLOG("creating transfer plugins ${basefile}_outplug.cmd and ${basefile}_inplug.cmd\n");
+open(PF, ">${basefile}_inplug.cmd") or die "Failed to create input transfer plugin : $!\n";
+if (is_windows()) {
+	print PF '@echo ' . $in_ad . ' > %~nx4' . "\n";
+} else {
+	print PF "#!/bin/sh\n";
+	print PF "echo '" . $in_ad . "' > " . '$4' . "\n";
+}
+close(PF);
+chmod 0744, "${basefile}_inplug.cmd";
+
+open(PF, ">${basefile}_outplug.cmd") or die "Failed to create output transfer plugin : $!\n";
+if (is_windows()) {
+	print PF '@echo ' . $out_ad . ' > %~nx4' . "\n";
+} else {
+	print PF "#!/bin/sh\n";
+	print PF "echo '" . $out_ad . "' > " . '$4' . "\n";
+}
+close(PF);
+chmod 0744, "${basefile}_outplug.cmd";
+
+my $num_ok = 0;
+my $on_success = sub {
+	my %info = @_;
+	my $jobid = $info{'cluster'} . '.' . $info{'job'};
+	TLOG("job $jobid completed successfully\n");
+	$num_ok += 1;
+};
+
+my $on_reconnectfailed = sub {
+	my %info = @_;
+	my $cluster_id = $info{'cluster'};
+	my $jobid = $cluster_id . '.' . $info{'job'};
+	TLOG("job $jobid failed to reconnect - aborting\n");
+	system("condor_rm $cluster_id");
+};
+
+my $result = SimpleJob::RunCheck(
+	queue_sz => 2,
+	runthis => "x_copy_binary_file.pl",
+	duration => "$inputdata result_file",
+	should_transfer_files => "YES",
+	transfer_input_files => "$inputdata, BIB://foo",
+	append_submit_commands => "BASE=$basefile"
+		. "\n" . 'transfer_output_remaps="result_file=BOB://bar"'
+		. "\n" . 'transfer_plugins = BOB=$(BASE)_outplug.cmd;BIB=$(BASE)_inplug.cmd'
+		,
+	on_success => $on_success,
+	on_reconnectfailed => $on_reconnectfailed,
+	);
+
+my $res = 0;
+$res = 1 if ($num_ok == 2);
+CondorTest::RegisterResult($res, test_name=>$testname, check_name=>'xfer-jobs-ran');
+
+TLOG("direct querying startd daemon ad to look for transfer totals\n");
+my @lines;
+# my $startd_addr = `condor_who -quick -af STARTD_Addr`;
+# chomp($startd_addr);
+# CondorTest::runCondorTool("condor_status -startd -direct \"$startd_addr\" -long", \@lines, 1, {emit_output => 1} );
+my $totals = `condor_who -startd -af AvgTransferInputMB TotalTransferInputMB AvgTransferOutputMB TotalTransferOutputMB`;
+
+# expected totals output is about 9.9 19.9 0.0011 0.0023
+# actual output will have more significant digits
+print "totals=$totals\n";
+print "expected (approx)=10.7 21.6 0.11 0.23\n";
```

Comment:
The first and last lines in this block don't seem to match?  Typo?

Reply:
comment is out of date.  will change if you want.

Reply:
Wouldn't be nice.  Would be safe to just remove entirely, I think?

Reply:
sure